app/components/channel_drawer/channels_list/channel_item/channel_item.js

```diff
@@ -81,6 +85,16 @@ export default class ChannelItem extends PureComponent {
             type
         } = this.props;
 
+        const {intl} = this.context;
+
+        let channelDisplayName = displayName;
+        if (currentUserId === channelTeammateId) {
+            channelDisplayName = intl.formatMessage({
+                id: 'mobile.channel.you',
```

Comment:
I think here you could just use `channel_header.directchannel.you` as it already exists and is the same message